test.js

```diff
@@ -92,6 +92,24 @@ describe('store', function () {
     store.has('a.b.c.d.x').should.be.true;
   });
 
+  it('should return true if a key `.hasOwn()` on the store', function () {
+    store = new Store('eee');
+    store.set('ggg', 'fff');
+    store.set('abc', null);
+    store.hasOwn('ggg').should.be.true;
+    store.hasOwn('abc').should.be.true;
+    store.hasOwn('foo').should.be.false;
+  });
+
+  it('should return true if a nested key `.hasOwn()` on the store', function () {
+    store = new Store('xxx');
+    store.set('a.b.c.d', {x: 'zzz'});
+    store.set('a.b.c.e', {f: null});
+    store.has('a.b.c.d.x').should.be.true;
+    store.has('a.b.c.e.f').should.be.true;
```

Comment:
I think one of these should show a different result than `hasOwn`, right?  meaning that if `store.has('a.b.c.e.f')` is true, given that the value is `null`, then is there a need for `hasOwn`? It seems that `.has` already checks keys, not values (sad but I don't remember, I go check in a little bit)